.factory/automation.yml

```diff
@@ -293,7 +293,7 @@ build:
 
         source tool/test/start-cloud-servers.sh 3 && # use source to receive export vars
           bazel test //python/tests/integration:test_connection --test_env=ROOT_CA=$ROOT_CA --test_output=streamed --jobs=1 &&
-          # TODO currently broken test
+          # TODO #635: currently broken test
```

Comment:
This format comes from [the short links format](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/autolinked-references-and-urls), but we can switch to full URLs if it's not that comfortable. I'm open to discussions.